File: Imperium/src/com/imperium/Commands/QueryAppConfigResponse.java
package com.imperium.Commands;

public class QueryAppConfigResponse implements ICommand{
    private CommandType type;
    private int length;
    private String json;

    public CommandType getType() {
        return type;
    }

    public int getLength() {
        return length;
    }

    public void setLength(int length) {
        this.length = length;
    }

    public String getJson() {
        return json;
    }

    public void setJson(String json) {
        this.json = json;
    }

    public QueryAppConfigResponse(){
        type = CommandType.QueryAppConfigResponse;
    }
}

File: Responder/Responder/Commands/QueryAppConfigResponse.cs
namespace Responder.AppConfig
{
    public class QueryAppConfigResponse : ICommand
    {
        #region Properties
        public CommandType Type { get; private set; }
        public int Length { get; set; }
        public string Json { get; set; }
        #endregion

        #region Constructors
        public QueryAppConfigResponse()
        {
            Type = CommandType.QueryAppConfigResponse;
        }
        #endregion
    }
}
